src/components/InstructorRevenueCard/index.js

```diff
@@ -23,95 +28,81 @@ export default ({revenueUrl}) => (
           return null
         }
 
-        const currentMonthDates = map(data, month => month.month)
+        const currentMonthNames = map(data, month => prettyMonthName(month.month))
```

Comment:
Made new `prettyMonthName` module that converts a `yyyy-mm-dd` string to `Jan`, `Feb` etc.:

![image](https://cloud.githubusercontent.com/assets/5497885/24728442/0f675178-1a17-11e7-9041-ea2b9dc9eb8d.png)